tests/smoke/CMakeLists.txt

```diff
@@ -65,6 +65,19 @@ if (ENABLE_FORTRAN AND ENABLE_FRUIT)
                  COMMAND blt_fruit_smoke)
 endif()
 
+###############
+# fruit mpi test
+###############
+if (ENABLE_FORTRAN AND ENABLE_FRUIT AND ENABLE_MPI AND ENABLE_FRUIT_MPI)
+    blt_add_executable(NAME blt_fruit_mpi_smoke
+                       SOURCES blt_fruit_mpi_smoke.f90
+                       OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+                       DEPENDS_ON fruit fruit_mpi MPI::MPI_Fortran
```

Comment:
`fruit_mpi` should be the only thing required here

Reply:
Done in [99322a95](99322a95fd471f4d729202d9fd4260758a8d0b44).